src/components/table-cell/table-cell-styles.jsx

```diff
@@ -63,7 +63,6 @@ export default theme => {
     },
     child: {
       display: 'block',
-      margin: [8, 0],
```

Comment:
Can't have padding on TD/TH due to flexbasis bug in IE11, see https://github.com/philipwalton/flexbugs#flexbug-7

I've tested and should be a margin: 8 on child and padding: 0 on the tablecell to get same effect